lib/mp4/transmuxer.js

```diff
@@ -638,13 +691,37 @@ Transmuxer = function(options) {
         audioSegmentStream = new AudioSegmentStream(audioTrack);
 
         // Set up the final part of the audio pipeline
-        aacStream
+        adtsStream
           .pipe(audioSegmentStream)
           .pipe(coalesceStream);
       }
     }
   });
 
+  };
+
+  Transmuxer.prototype.init.call(this);
+  options = options || {};
+  aacfile = options.aacfile;
```

Comment:
Does this need to be a variable? You can just do `option.aacfile === undefined` in the one place it is used. Also consider naming this something like `aacInput`.